packages/components/field-label/src/field-label.js

```diff
@@ -18,6 +18,10 @@ export const FieldLabel = (props) => {
       `ui-kit/FieldLabel: setting an hintIcon size is not supported.`
     );
   }
+  const a11yLabelProps = {
+    ...(props.id ? { id: props.id } : {}),
+    ...(props.htmlFor ? { htmlFor: props.htmlFor } : {}),
+  };
```

Comment:
Is this capture needed? Can't we just pass the props below? 🤔

Reply:
Hmm probably not. You can't pass both, that's why I added the invariant. I think I forgot to remove this.
Thanks for the reminder.